p4-16/spec/P4-16-spec.mdk

```diff
@@ -1496,12 +1495,21 @@ Each parameter may be labeled with a direction:
   call, the value of the parameter is copied to the corresponding storage location
   for that l-value.
 - `inout` parameters are both `in` and `out`. An argument passed as
-    an `inout` parameter must be an l-value.
-- No direction indicates that value of parameter is either:
-	- a compile-time known value
-	- an action parameter that can only be set by the control plane
-	- an action parameter that can be set directly by another calling
-      action; in this case it behaves like an `in` parameter
+  an `inout` parameter must be an l-value.
+- The meaning of parameters with no direction depends upon the kind of
+  entity the parameter is for:
+  - For anything other than an action, e.g. a control, parser, or
+    function, a directionless parameter means that the value supplied
+    as an argument in a call must be a compile-time known value.
```

Comment:
@jafingerhut Add a cross-reference to the existing spec section describing what "compile-time known value" is.

Reply:
Added cross-reference.